[2024-01-05T21:08:26.224+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.get_data scheduled__2024-01-01T13:30:00+00:00 [queued]>
[2024-01-05T21:08:26.231+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.get_data scheduled__2024-01-01T13:30:00+00:00 [queued]>
[2024-01-05T21:08:26.231+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-05T21:08:26.231+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-05T21:08:26.231+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-05T21:08:26.239+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): get_data> on 2024-01-01 13:30:00+00:00
[2024-01-05T21:08:26.241+0300] {standard_task_runner.py:55} INFO - Started process 76530 to run task
[2024-01-05T21:08:26.246+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'get_data', 'scheduled__2024-01-01T13:30:00+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmpm1prbrad']
[2024-01-05T21:08:26.247+0300] {standard_task_runner.py:83} INFO - Job 120: Subtask get_data
[2024-01-05T21:08:26.344+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.get_data scheduled__2024-01-01T13:30:00+00:00 [running]> on host prod-srv
[2024-01-05T21:08:26.428+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T13:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T13:30:00+00:00
[2024-01-05T21:08:26.429+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-05T21:08:26.429+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/igor/mlops_4/scripts/get_data.py']
[2024-01-05T21:08:26.436+0300] {subprocess.py:86} INFO - Output:
[2024-01-05T21:08:27.123+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-05T21:08:27.168+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=get_data, execution_date=20240101T133000, start_date=20240105T180826, end_date=20240105T180827
[2024-01-05T21:08:27.220+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-05T21:08:27.247+0300] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
